src/codec_rav1e.c

```diff
@@ -3,7 +3,7 @@
 
 #include "avif/internal.h"
 
-#include "rav1e.h"
+#include "rav1e/rav1e.h"
```

Comment:
rav1e header is installed to `<prefix>/include/rav1e/rav1e.h`, so this seems more standard.

Reply:
I re-examined the situation.
rav1e's pkg-config file set additional include path to `${prefix}/include/rav1e`, instead of the more common `${prefix}/include`, so despite placed at `${prefix}/include/rav1e/rav1e.h`, it is included by using `#include "rav1e.h"`. `#include "rav1e/rav1e.h"` works probably just because other dependency added include path `${prefix}/include`. I'm reverting these changes.